playground/Lambda/Lambda.AppHost/Program.cs

```diff
@@ -35,10 +36,10 @@
 
 builder.AddAWSLambdaFunction<Projects.SQSProcessorFunction>("SQSProcessorFunction", "SQSProcessorFunction::SQSProcessorFunction.Function::FunctionHandler")
         .WithReference(awsSdkConfig)
-        .WithSQSEventSource(sqsDemoQueue)
+        .WithSQSEventSource(sqsDemoQueue1)
         // This reference is not necessary. It is added to confirm duplicate
         // CDK output parameters are not attempted to be added.
-        .WithReference(sqsDemoQueue);
+        .WithReference(sqsDemoQueue2);
```

Comment:
why did we remove sqsDemoQueue from here, wasnt that testing the other scenario before? shouldnt it be both 

`.WithReference(sqsDemoQueue`) // duplicate
`.WithReference(sqsDemoQueue2);` // not duplicate

Reply:
Good call. I added a reference for both demo1 and demo2 queues to confirm the behavior. In the future I would like to add another project to the playground that would act as the SQS publisher instead of adding these kind of random references.